src/components/Select.jsx

```diff
@@ -1,29 +1,30 @@
-"use client";
-import { useState } from "react";
-import styles from "../partials/_select.module.scss";
+'use client';
+import { useState } from 'react';
+import styles from '@/partials/_select.module.scss';
 
 export default function Select() {
   const [open, setOpen] = useState(false);
-  const [selected, setSelected] = useState("to do");
-  const [array, setArray] = useState(["to do", "doing", "done"]);
```

Comment:
Don't do this. Please be as descriptive with your variables as possible to avoid confusion for your future self reading back on this code.